src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/CourseAreaRepository.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.helper
+
+import org.springframework.data.repository.CrudRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.CourseActivityAreas
+
+@Repository
+interface CourseAreaRepository : CrudRepository<CourseActivityAreas, Long>
```

Comment:
V minor but these repos normally have the full entity name, ie CourseActivityAreaRepository. I know this one is only for test 🤔